gigabyte-ampere-cuttlefish-installer/README.md

```diff
@@ -0,0 +1,55 @@
+# Gigabyte Ampere Cuttlefish Installer
```

Comment:
Could we change it to a more universal name since it's not exclusive to Gigabyte?

Reply:
same for the project directory name

Reply:
Changed to cuttlefish-host-image-installer.